packages/openapi3/src/schema-emitter.ts

```diff
@@ -840,7 +842,30 @@ export class OpenAPI3SchemaEmitter extends TypeEmitter<
       };
     }
 
-    if (shouldInline(this.emitter.getProgram(), type)) {
+    const checkNeedForceRef = (
```

Comment:
should be part of `shouldInline`?

Reply:
no, the scenario is very special,it is only handled when `array of model` and `response body`